Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -414,19 +429,12 @@ public class ChatChannelController: DataController, DelegateCallable, DataStoreP
         let limit = limit ?? channelQuery.pagination?.pageSize ?? .messagesPageSize
         channelQuery.pagination = MessagesPagination(pageSize: limit, parameter: .lessThan(messageId))
 
-        isLoadingPreviousMessages = true
```

Comment:
Removing this handling would simplify things a lot 👍 We just need to make sure nothing is broken.